ntpcheck/checker/chrony.go

```diff
@@ -181,6 +176,17 @@ func (n *ChronyCheck) ServerStats() (*ServerStats, error) {
 	return serverStats, nil
 }
 
+// Unix returns true if connected via a unix socket
+func (n *ChronyCheck) Unix() bool {
+	// it could be a mock, so verify type assertion
+	if client, ok := n.Client.(*chrony.Client); ok {
+		if _, ok := client.Connection.(*chronyConn); ok {
+			return true
```

Comment:
Shouldn't it be:
```
if v, ok := client.Connection.(*chronyConn); ok {
			return v.local
```
?

Reply:
`chronyConn` always represents a unix socket connection, so this is enough